src/less/pagination/ds6/pagination.less

```diff
@@ -15,23 +16,17 @@
 }
 
 .pagination__items {
-    overflow: hidden;
-    white-space: nowrap;
-}
-
-.pagination__items > li:not([hidden]) {
-    margin-right: 8px;
-}
-
-.pagination__items > li:last-child {
-    margin-right: 0;
+    > li:not([hidden]) {
+        width: 56px;
```

Comment:
Could we just make them all 56px, whether they're visible or not? Does that affect anything?

Reply:
this is to override a style from the base/pagination.less.  I wasn't sure if that was being utilized differently for ds4.  If this isn't needed for ds4 I propose modifying the base/pagination and make it just 
```
> li { and remove the :not([hidden]);
```